### DISCUSSION TITLE
Publish passphrasemeter to Maven Central?

### DISCUSSION THREAD

**Discussion by CamiloCon** [wultra/wultra-docs#17] [Q&A] [answered]
Since jcenter will be maintained as a read-only repository indefinitely. Are there any plans to move `passphrasemeter` away from it? Is it published only in jcenter? Maven central sounds like a good alternative.

**Comment by petrdvorak** [comment, marked as answer]
Hello @CamiloCon, thank you for the report - this seems to be more a discussion than an issue. Generally speaking, we do publish all new library versions to Maven Central. We keep existing releases in JCenter, which will remain read-only indefinitely. So the library updates will move from JCenter to Maven Central.
